Directory.Build.props

```diff
@@ -67,7 +67,7 @@
 
   <!-- Package references which are consumed by all projects -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.Net.Compilers.Toolset" IsImplicitlyDefined="true" />
+    <PackageReference Include="Microsoft.Net.Compilers.Toolset" IsImplicitlyDefined="true" PrivateAssets="all" />
```

Comment:
Needed to mark this as `PrivateAssets="all"` so that the product packages don't list it as a dependency.